README.md

```diff
@@ -269,6 +272,55 @@ $ ./gradlew check
 $ cd react-client/
 $ npm test
 ```
+
+## End to end
+
+### via `run-on-data` GitHub workflow job
+
+There is a way to locally execute the [`run-on-data` job of the end_to_end GitHub workflow](https://github.com/MobilityData/gtfs-validator/blob/24d58c8ee76af00aa3ab413b218b4a8e2cfafc4b/.github/workflows/end_to_end.yml#L9)
+
+You need Docker installed
```

Comment:
I couldn't add a suggestion. However this could be "You need Docked to be installed"